tests/unit_tests/test_resources.py

```diff
@@ -92,6 +93,18 @@ def test_kubernetes_labels_resources():
     _run_label_test(allowed_labels, invalid_labels, cloud)
 
 
+def test_no_cloud_labels_resources():
```

Comment:
If we are using `enabled_clouds` above, we can add a fixture in this function to make sure we mock the enabled clouds for unittest: `def test_no_cloud_labels_resources(enable_all_clouds):`